app/models/agents/digest_agent.rb

```diff
@@ -10,6 +10,8 @@ class DigestAgent < Agent
       The resulting Event will have a payload message of `message`. You can use liquid templating in the `message`, have a look at the [Wiki](https://github.com/cantino/huginn/wiki/Formatting-Events-using-Liquid) for details.
 
       Set `expected_receive_period_in_days` to the maximum amount of time that you'd expect to pass between Events being received by this Agent.
+
+      If `clear_queue` is set to 0, Events are automatically purged from the memory when an Event is emitted by this Agent. To maintain a fixed number of Events, set `clear_queue` to that number.
```

Comment:
I am unsure myself, and tried similar naming schemes to the ones you mentioned. Maybe `Maintain X events for digest`. I'll think more on this.